content/blog/2016/2016-08-30-ask-experts-demos.adoc

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title: "Demos at Jenkins World 2016"
+tags:
+- jenkinsworld
+- event
+author: rtyler
+---
+
+image:/images/conferences/Jenkins-World_125x125.png[role=right]
+
+At this year's link:https://www.cloudbees.com/jenkinsworld/home[Jenkins World],
+our events officer link:https://github.com/alyssat[Alyssa] has been working to
+organize various activities in the "Open Source Hub" on the expo floor. One of
+the activities that I am personally most excited for are the Jenkins demos in
+our "open source theater." Both days of the conference (Sept. 14th and 15th),
+during the break for lunch, there will be 15 minute demos by many of the
+link:/blog/2016/08/26/ask-the-experts-jenkins-world/[experts] helping to staff
+the Open Source Hub.
+
+== Demo Schedule
+
+=== Wednesday, September 14th
+[options=header]
+|===
+| Time | Session | Details | Presenter
+| 12:15 - 12:30
+| *Blue Ocean in Action*
+| _Showcase of Blue Ocean and how it will make Jenkins a pleasure to use._
```

Comment:
Might be better to clarify what Blue Ocean is, for those who don't already know.